_sass/scaffolds.scss

```diff
@@ -164,7 +164,7 @@ label {
   display: inline-block;
   width: 1.4em;
   height: 1.4em;
-  line-height: 1.5; 
+  line-height: 1.5;
```

Comment:
css rules actually require a unit here, i think we want `em` so `line-height: 1.5em;`